docs/content/introduction/intro-invocation-images.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Building Invocation Images
+title: Invocation Images
```

Comment:
```suggestion
title: Bundle Installers
```
[Carolyn wanted this change awhile back](https://github.com/getporter/porter/issues/1188) and honestly I can't think of a better name